### PR TITLE
Bad Design is Bad for Your Health: Why Data Visualization Details Matter

### PR DESCRIPTION
**Pull request by jakekramercerner**

### CODE REVIEW

**Comment by anelson425** [comment]
bad design is bad for your health_ why_ - http___localhost_4000_blog_bad-desi

**Comment by anelson425** [comment]
Research

**Comment by anelson425** [comment]
Accidentally closed so I reopened.

File: source/_posts/2018-01-04-bad-design-is-bad-for-your-health-why-data-visualization-details-matter.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Qualitative and quantitative data is visually encoded, or translated, through a variety of graphical methods that make up each visualization type. [Research](https://ils.unc.edu/courses/2015_spring/inls541_001/Readings/Cleveland%20and%20McGill%201985%20-%20Graphical%20Perception%20and%20Cleveland1985-Graphical%20Methods%20for%20Analyzing%20Scientific%20Data.pdf) has been done on these visual encoding methods, giving us insights for human graphical perception. Statistician William Cleveland and his colleague, Robert McGill, tested the visual encoding methods and ranked them by accuracy of perception. [Their findings](https://ils.unc.edu/courses/2015_spring/inls541_001/Readings/Cleveland%20and%20McGill%201985%20-%20Graphical%20Perception%20and%20Cleveland1985-Graphical%20Methods%20for%20Analyzing%20Scientific%20Data.pdf) give us a scientific foundation for data visualization.
```
Comment (anelson425) [inline source/_posts/2018-01-04-bad-design-is-bad-for-your-health-why-data-visualization-details-matter.markdown]: The reference to the research link exists twice here, probably only need to include it once.

Comment (jakekramercerner) [reply source/_posts/2018-01-04-bad-design-is-bad-for-your-health-why-data-visualization-details-matter.markdown]: Fixed, thanks :)

**Pull request merged**